pytest_splunk_addon/standard_lib/app_test_generator.py

```diff
@@ -79,9 +79,10 @@ def generate_tests(self, fixture):
                 self.cim_test_generator.generate_tests(fixture), fixture
             )
         elif fixture.startswith("splunk_searchtime_requirement"):
-            yield from self.dedup_tests(
-                self.requirement_test_generator.generate_tests(fixture), fixture
-            )
+            if self.pytest_config.getoption("requirement_test"):
```

Comment:
should it be `requirement_test` or `requirement-test` like in [confluence](https://confluence.splunk.com/pages/viewpage.action?pageId=335119033#Requirementtests-RunrequirementtestswithPytestSplunkAddon) and [pytest.ini](https://github.com/splunk/pytest-splunk-addon/blob/7bdace54b64ce095e78df6902ac78a9410eef1f3/tests/test_splunk_addon.py#L496) ?

Reply:
dest is requirement_test it in https://github.com/splunk/pytest-splunk-addon/blob/44e974419511f0a43f118938485789c0895cb434/pytest_splunk_addon/splunk.py#L251 
 ```